prime-dotnet-webapi/Controllers/EnrolleesController.cs

```diff
@@ -622,5 +623,46 @@ public async Task<ActionResult> DeleteFutureEnrolleeAbsence(int enrolleeId, int
 
             return NoContent();
         }
+
+        // POST: api/enrollees/1/absences/email
+        /// <summary>
+        ///    Sends Enrollee Absence Notification Email
+        /// </summary>
+        /// <param name="enrolleeId"></param>
+        /// <param name="email"></param>
+        [HttpPost("/api/enrollees/{enrolleeId}/absences/email", Name = nameof(SendEnrolleeAbsenceEmail))]
+        [Authorize(Roles = Roles.PrimeEnrollee)]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        public async Task<ActionResult> SendEnrolleeAbsenceEmail(int enrolleeId, FromBodyText email)
+        {
+            if (!Email.IsValidEmail(email))
+            {
+                return BadRequest("The email provided is not valid.");
+            }
+
+            var enrollee = await _enrolleeService.GetEnrolleeAsync(enrolleeId);
```

Comment:
rather than getting the whole enrollee, you can use `EnrolleeService.GetPermissionsRecordAsync()`